Library/.rubocop.yml

```diff
@@ -327,6 +327,11 @@ Style/Documentation:
     - "**/{Formula,Casks}/**/*.rb"
     - "**/*.rbi"
 
+# This is impossible to fix if the line exceeds the maximum length.
```

Comment:
It feels like this is maybe an upstream issue?

Reply:
[When it was last raised](https://redirect.github.com/rubocop/rubocop/pull/10742), they chose to disable autocorrection under that scenario and document it: https://docs.rubocop.org/rubocop/cops_style.html#styleemptymethod. Which is weird because there's no manual fix either.